Rakefile.rb

```diff
@@ -48,6 +48,36 @@ def name_from(json)
   orgs.first[:name]
 end
 
+def file_to_commit_metadata
+  # There's about 2MB of data returned by this command, so rather than
+  # using backticks to turning it into a string and then splitting it,
+  # parse the output line-by-line. As a bonus this doesn't
+  # unnecessarily invoke a shell.
+  command = ['git', '--no-pager', 'log', '--name-only',  '--format=%h|%at', '--', 'data']
+  result = {}
+  sha, timestamp = nil
```

Comment:
Oops, yes. I like the monkey-patch (apart from my usual reservation about never being clear how to be sure if some other code is patching it differently :)), so I've added that - thanks for the suggestion. The fixup commit is: 010cf76b979aa29c25d785